src/geo/leaflet/vector.js

```diff
@@ -0,0 +1,95 @@
+var cdb = window.cdb;
+var L = cdb.L;
+var _ = cdb._;
+var util = cdb.core.util;
+var LeafletLayerView = cdb.geo.LeafletLayerView;
+var CartoDBLogo = cdb.geo.common.CartoDBLogo;
+
+var LeafletVectorLayer = L.CartoDBd3Layer.extend({
+
+  initialize: function(layerModel, leafletMap) {
+    var extra = layerModel.get('extra_params');
+
+    var query = this._getQuery(layerModel);
+
+    // initialize the base layers
+    L.CartoDBd3Layer.prototype.initialize.call(this, {
+      table: layerModel.get('table_name'),
+      user: layerModel.get('user_name'),
+      blendmode: layerModel.get('torque-blend-mode'),
+      sql_api_domain: layerModel.get('sql_api_domain'),
+      sql_api_protocol: layerModel.get('sql_api_protocol'),
+      sql_api_port: layerModel.get('sql_api_port'),
+      tiler_protocol: layerModel.get('tiler_protocol'),
+      tiler_domain: layerModel.get('tiler_domain'),
+      tiler_port: layerModel.get('tiler_port'),
+      maps_api_template: layerModel.get('maps_api_template'),
+      stat_tag: layerModel.get('stat_tag'),
+      animationDuration: layerModel.get('torque-duration'),
+      steps: layerModel.get('torque-steps'),
+      sql: query,
+      visible: layerModel.get('visible'),
+      extra_params: {
+        api_key: extra ? extra.map_key: ''
+      },
+      cartodb_logo: layerModel.get('cartodb_logo'),
+      attribution: layerModel.get('attribution'),
+      cartocss: layerModel.get('cartocss') || layerModel.get('tile_style'),
+      named_map: layerModel.get('named_map'),
+      auth_token: layerModel.get('auth_token'),
+      no_cdn: layerModel.get('no_cdn'),
+      dynamic_cdn: layerModel.get('dynamic_cdn'),
+      loop: layerModel.get('loop') === false? false: true,
+      instanciateCallback: function() {
+        var cartocss = layerModel.get('cartocss') || layerModel.get('tile_style');
+
+        return '_cdbct_' + cdb.core.util.uniqueCallbackName(cartocss + query);
```

Comment:
Use another name to not collide with torque's one (very unlikely due different cartocss most of the time).